product_docs/docs/pgd/5/admin-tpa/upgrading_major_rolling.mdx

```diff
@@ -0,0 +1,510 @@
+---
+title: Performing a Postgres major version rolling upgrade 
```

Comment:
Missing a navTitle and add deepToC: True to get more than two headings in the right hand nav bar

Reply:
Thanks!